app/views/assessments/show.html.erb

```diff
@@ -28,37 +28,25 @@
     <%# Display any optional instructor admin options %>
     <% if @cud.instructor? then %>
       <h3>Admin Options</h3>
-      <ul class="options">
-    
-        <li> <%= link_to "Edit assessment", {:action => "edit" }, {:title=> "View and modify the properties for this assessment, including its problems" } %></li>
+      <ul class="options">    
         <li> <%= link_to "Grade submissions", {:action => "viewGradesheet" }, {:title=> "View and enter grades on the gradesheet" } %></li>
         <li> <%= link_to "Release all grades", {:action => "releaseAllGrades" }, {:title=>  "Make all scores for this assessment visible to students" } %></li>
         <li> <%= link_to "Withdraw all grades", {:action => "withdrawAllGrades" }, {:title=>  "Hide all scores for this assessment from students" } %></li>
         <li> <%= link_to "Export assessment", {:action => "export" }, {:title=> "Export persistent properties that will be imported when you install this assessment in a future term" } %></li>
-        <li> <%= link_to "Reload config file", {:action => "reload" }, {:title=> "Reload the assessment config file (provided for backward compatibility with legacy assessments)" } %></li>
         <li> <%= link_to "Manage extensions", [@course, @assessment, :extensions], { title: "Give extensions to students" } %></li>
         <li> <%= link_to "Manage submissions", [@course, @assessment, :submissions], { title: "Create, view, export, and re-autograde submissions" } %></li>
         <li> <%= link_to "Manage assessment attachments", [@course, @assessment, :attachments], { title: "Distribute files to your students" } %></li>
         <li> <%= link_to "View statistics", {:action => "statistics" }, {:title=> "View detailed stats for this assessment" } %></li>
         <li> <%= link_to "Bulk import grades", {:action => "bulkGrade" }, {:title=> "Upload scores or feedback for multiple students from a CSV file" } %></li>
         <li> <%= link_to "Bulk export grades", {:action => "bulkExport" }, {:title=> "Export grades (with sub-scores) \nfor all students to a CSV file" } %></li>
-
         <% if @assessment.has_svn then %>
-          <li><%= link_to "SVN Settings", [:admin_svn, @course, @assessment], title: "Manage the SVN repositories" %></li>
+        <li> <%= link_to "Admin SVN", [:admin_svn, @course, @assessment], title: "Manage the SVN repositories" %></li>
```

Comment:
I guess I see why this isn't quite a "Settings" page, but I don't really like the use of "Admin" as a verb either.  Maybe "Manage SVN" would be better?  Same for groups.